packages/ingest/url-path-template.js

```diff
@@ -0,0 +1,17 @@
+const { template } = require('lodash');
+
+/**
+* define the path of a file based on the metadata of a granule
+*
+* @param {string} pathTemplate - the template that defines the path,
+* using `{}` for string interpolation
+* @param {Object} context - the metadata used in the template
+* @returns {string} - the url path for the file
+**/
+module.exports = function urlPathTemplate(pathTemplate, context) {
```

Comment:
Write unit tests for this function

Reply:
I'm going to move this function and unit tests to a different pr as it's not actually used in these changes.